packages/vite/src/node/config.ts

```diff
@@ -144,6 +144,11 @@ export interface UserConfig {
    * Preview specific options, e.g. host, port, https...
    */
   preview?: PreviewOptions
+  /**
+   * Force dep pre-optimization regardless of whether deps have changed.
+   * TODO: Should it be optimizeDeps.force?
+   */
+  force?: boolean
```

Comment:
Do you mean instead of `config.force`? This option is to delete the deps cache, same as in dev (we had it in `config.server.force` before).

Reply:
We talked about this, `force` doesn't mean much. We are going to mark it as @experimental to help people test since it is the CLI option everybody knows about, but we need to discuss and find a better name.